zkevm-circuits/src/evm_circuit/execution/calldatacopy.rs

```diff
@@ -135,7 +135,7 @@ impl<F: Field> ExecutionGadget<F> for CallDataCopyGadget<F> {
                 src_addr,
                 src_addr_end,
                 memory_address.offset(),
-                bytes_length_word.expr(),
```

Comment:
remove bytes_length_word declare and assignment  code in each copy related opcode.